config/_default/config.toml

```diff
@@ -46,18 +46,21 @@ hasChildren = true
   [[languages.en.menu.main]]
   parent = "Scenario"
   name = "DevOps"
+  Title = "Embracing One-stop DevOps Workflow"
   URL = "dev-ops"
```

Comment:
Please change the file name and url path from dev-ops to devops.
![image](https://user-images.githubusercontent.com/40452856/84476811-44bb1b80-acc1-11ea-8a78-e07fef73382d.png)